### PR TITLE
Add Custom Config merger for Kafka - 3.3.2

### PR DESCRIPTION
**Pull request by obaydullahmhs**
Signed-off-by: obaydullahmhs <obaydullah@appscode.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,24 +10,37 @@ LABEL name="kafka" version=${KAFKA_VERSION}
...
+ && apt-get -y install kafkacat \
  && wget -O /tmp/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz https://archive.apache.org/dist/kafka/${KAFKA_VERSION}/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz \
  && tar xfz /tmp/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz -C /opt \
  && rm /tmp/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz \
  && mv /opt/kafka_${SCALA_VERSION}-${KAFKA_VERSION} ${HOME} \
  && rm -rf /opt/kafka_${SCALA_VERSION}-${KAFKA_VERSION}.tgz
 
-COPY ./entrypoint.sh /opt/kafka/config
+COPY scripts/entrypoint.sh /opt/kafka/config
+COPY scripts/merge_custom_config.sh /opt/kafka/config
```
Comment (raihankhan) [inline Dockerfile]: can we combine them in a single line?
- `COPY scripts /opt/kafka/config`

File: Dockerfile [hunk 2]
```diff
@@ -10,24 +10,37 @@ LABEL name="kafka" version=${KAFKA_VERSION}
 LABEL org.opencontainers.image.source https://github.com/kubedb/kafka-docker
 
 
-# https://archive.apache.org/dist/kafka contains all the kafka version binary
+# ref: https://archive.apache.org/dist/kafka contains all the kafka version binary
+# ref: https://github.com/edenhill/kcat used for installing kafkacat cli
 RUN apt-get update \
- && apt-get install wget \
+ && apt-get install -y wget sudo \
+ && apt-get -y install kafkacat \
```
Comment (raihankhan) [inline Dockerfile]: change it to - `apt-get install -y wget sudo kafkacat`
also add a small comment to explain why we are installing sudo

**Review by raihankhan — COMMENTED** [review]
Please make these changes in other PRs also.

**Review by raihankhan — APPROVED** [review]
LGTM!

**Review by heheh13 — APPROVED** [review]

**Pull request merged**